mlir-sycl/test/Dialect/IR/SYCL/constructor.mlir

```diff
@@ -1,35 +0,0 @@
-// RUN: sycl-mlir-opt -allow-unregistered-dialect %s -split-input-file | FileCheck %s
```

Comment:
We will store all tests in `ops.mlir`

Reply:
Oh, I see what you mean now. I don't think it's worth it testing that here. Our definition/verification is not extensive enough anyway.